operator/scheduler/experiment.go

```diff
@@ -76,25 +76,15 @@ func (s *SchedulerClient) StopExperiment(ctx context.Context, experiment *v1alph
 func (s *SchedulerClient) SubscribeExperimentEvents(ctx context.Context, grpcClient scheduler.SchedulerClient, namespace string) error {
```

Comment:
I quite like that these functions now end up less entangled with code reconciling the state of the scheduler